DSharpPlus/Entities/DiscordGuildMembershipScreeningField.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace DSharpPlus.Entities
+{
+    /// <summary>
+    /// Represents a field in a guild's membership screening form
+    /// </summary>
+    public class DiscordGuildMembershipScreeningField
+    {
+        /// <summary>
+        /// Gets the type of the field. Currently defualts to TERMS.
+        /// </summary>
+        [JsonProperty("field_type")]
+        public string Type { get; internal set; }
```

Comment:
Part of me is also debating making this an enum, though I'm not sure because Discord decided to make this a string for some reason. Outside opinions are welcome.

Reply:
Yeah I think we'll want to make this an enum instead. Newtonsoft has a way to serialize an enum from a string, see [LavalinkTrack](https://github.com/DSharpPlus/DSharpPlus/blob/b89a6486bf064ea9bf4b959bedb22f4f79303be1/DSharpPlus.Lavalink/LavalinkTrack.cs#L75)